site/docs/v1/tech/guides/configuring-application-specific-email-templates.adoc

```diff
@@ -0,0 +1,91 @@
+---
+layout: doc
+title: Configure Application Email Templates
+description: I want to use application specific email templates to override Tenant level templates
+navcategory: developer
+---
+
+:page-liquid:
+
+:sectnumlevels: 0
+
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+== Application Specific Email Templates
+
+Before you begin, complete the following guides:
+
+    * link:docs/v1/tech/getting-started/_5-minute-setup-guide.adoc[5 minute setup guide]
+    * link:docs/v1/tech/email-templates/configure-email[Configure Email]
+    * link:/docs/v1/tech/core-concepts/applications[Application overview]
+    * link:docs/v1/tech/tutorials/register-user-login-api[Register a user and login]
+
+In FusionAuth, a Tenant can be configured to send transactional emails using default Email Templates that ship with FusionAuth, such as `Email Update`, `Forgot Password`, `Suspicious login`, etc. See the full list of email templates that can be configured here: link:docs/v1/tech/core-concepts/tenants#template-settings[Template Settings]
+
+To begin, we'll configure our email template at the Tenant level to ensure they are sent when a user starts the link:docs/v1/tech/apis/users[Start Forgot Password Workflow].
+
+Next, navigate to [breadcrumb]#Tenant -> Email -> Template Settings -> Forgot Password#. Select `Forgot Password` from the dropdown, then click the blue save icon. When a user clicks the `Forgot Password` link on the hosted login page or calls the link:/docs/v1/tech/apis/users#start-forgot-password-workflow[forgot password API], this template will be used to email the user to start the workflow to change their password for all applications in the specified Tenant - so long as they don't have a template set at the Application level.
+
+Make sure you've created an Application in the link:/docs/v1/tech/core-concepts/applications[Application overview] guide. After, log in to your application by clicking on `Applications` in the navigation sidebar, select the green magnifying glass icon, and copy and paste the `Login URL` into an incognito browser. Click the Forgot Password link, enter your email in the form, and you should see the `Forgot Password` email sent to your inbox.
+
+### Overriding email templates
+
+Finally, we'll override our generic Tenant Email Template we just configured with a custom application specific template for a fictitious company called Pied Piper.
+
+Navigate to [breadcrumb]#Customizations -> Emails Templates# and click the green + icon to create a new template.
+Fill out the `Name`, `Default Subject`, `From Email`, `Default from Name` fields.
+Paste the below code snippet into the `HTML Template` section and modify based on your application:
```

Comment:
I might say "This template is modified to reference a certain application. Feel free to modify this example as needed."